client/app/components/partials/FilesDialog.vue

```diff
@@ -513,8 +513,10 @@ export default {
     },
     onModeChanged: function() {
       if (this.mode == 'trio' && this.cohortModel.getCanonicalModels().length < 3 ) {
+        this.$emit('isTrio', true);
```

Comment:
If the `trio` option is selected on the files dialog when uploading the data, it sets the flag `isTrio` as true and is passed across the components.